src/core/api/index.js

```diff
@@ -55,7 +55,7 @@ export function callApi({ endpoint, schema, params = {}, auth = false, state = {
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-export function search({ api, page, query }) {
+export function adminSearch({ api, page, query }) {
```

Comment:
That would mean `auth = true` implicitly changes the API URL endpoint. I'm fine with that, so if you are too sounds good 👍